openlayers/openlayers.d.ts

```diff
@@ -12619,7 +12619,7 @@ declare module olx {
          *     visible: (boolean|undefined)}}
          */
         interface VectorOptions {
-            renderOrder: (feature1: ol.Feature, feature2: ol.Feature) => number;
+            renderOrder?: (feature1: ol.Feature, feature2: ol.Feature) => number;
```

Comment:
There is a documentation conflict here. The [documentation ](http://openlayers.org/en/latest/apidoc/ol.layer.Vector.html) text explicitly says this argument is required and that you can "use null to avoid the sort, but get an undefined draw order". However the field definition says it can have be `undefined`. 
I quickly checked the code and it seems to properly handle the `undefined` case. So the modification should be OK.